kitsune/users/forms.py

```diff
@@ -175,11 +177,17 @@ def clean(self):
 class ProfileForm(forms.ModelForm):
     """The form for editing the user's profile."""
 
+    involved_from = forms.DateField(
+        required=False,
+        label=_lazy(u'Involved with SUMO from'),
+        widget=MonthYearWidget(years=range(1998, datetime.today().year + 1),
```

Comment:
How did you come up with the year 1998? It is probably worth adding a comment with that info.